### PR TITLE
API 4493: upgrade webpack to latest version of 4

### PR DESCRIPTION
**Pull request by anthony-kopczyk**
### Description

This upgrades our webpack to the latest possible version we can get working, which happens to be the last released version of 4.

Version 5 does not work properly with the latest version of react-dev-utils at this time
- https://github.com/facebook/create-react-app/issues/9880

### Process

- [N/A] Tests added or updated for change
- [N/A] Docs updated
  - [N/A] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [N/A] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [N/A] Accessibility concerns have been considered and addressed
  
  - [N/A] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### Requested feedback

The main thing is to double check everything appears to still be running properly.

### CODE REVIEW

File: SitemapBuilderWebpackPlugin.js [hunk 1]
```diff
@@ -39,12 +39,13 @@ class SitemapBuilderPlugin {
   }
 
   apply(compiler) {
-    compiler.plugin('emit', (compilation, callback) => {
+    compiler.hooks.emit.tapAsync('SitemapBuilderPlugin', (compilation, callback) => {
```
Comment (anthony-kopczyk) [inline SitemapBuilderWebpackPlugin.js:42]: This is deprecated in webpack 4, so I updated it. In webpack 5 the removed version will no longer be available. This is the only deprecated feature I saw us currently using.

**Comment by anthony-kopczyk** [comment]
Moving this into review, but as an FYI, I think we will be waiting to merge until https://github.com/department-of-veterans-affairs/developer-portal/pull/627 is merged.

**Review by johanna-adhoc — APPROVED** [review]
Tested this in local and it passed all unit test, eslint and prod build. LGTM!

**Review by johanna-adhoc — APPROVED** [review]
LGTM

**Comment by anthony-kopczyk** [comment]
For anyone looking at the diff on this at a later date, we had a coincidence where another PR updated webpack to the latest version of 4, but this still included an update to webpack-cli and removed a deprecated webpack feature.

**Review by TS000 — APPROVED** [review]
Pulled down and did unit tests and regression test suites. Also checked things over on Desktop and Mobile. Looks great!

**Pull request merged**